envoy/api/v2/cds.proto

```diff
@@ -356,6 +356,19 @@ message Cluster {
     // .. note::
     //   The specified percent will be truncated to the nearest 1%.
     core.Percent healthy_panic_threshold = 1;
+    // [#not-implemented-hide:]
+    // Configures percentage of requests that will be routed to same zone
+    // If not specified, the default is 100%.
+    // :ref:`runtime values <config_cluster_manager_cluster_runtime_zone_routing>`.
```

Comment:
Did we regress here? I thought we were going to put `*` before each of the links to turn them into bullet points, here and below?